docs/GettingStarted.md

```diff
@@ -1169,6 +1170,8 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | `quantize.query.exclude` | Defines which values should be removed entirely. Excludes nothing by default. May be an Array of strings, or `:all` to remove the query string entirely. Option must be nested inside the `query` option. | `nil` |
 | `quantize.fragment` | Defines behavior for URL fragments. Removes fragments by default. May be `:show` to show URL fragments. Option must be nested inside the `quantize` option. | `nil` |
 | `request_queuing` | Track HTTP request time spent in the queue of the frontend server. See [HTTP request queuing](#http-request-queuing) for setup details. Set to `true` to enable. | `false` |
+| `rum_injection_enabled` | Connect frontend traces from the RUM (real user monitoring) [`browser-sdk`](https://docs.datadoghq.com/real_user_monitoring/installation/?tab=us) to backend traces by automatically injecting an HTML Comment containing the datadog trace-id. See [RUM Injection](#rum-injection) for setup details. Set to `true` to enable. Environment Variable: `DD_RUM_INJECTION`. *Experimental* | `false` |
+| `rum_cached_pages` | Define which pages to exclude from automatically injecting an HTML Comment containing the Datadoog trace-id.  See [RUM Injection](#rum-injection) for setup details. Accepts an array of path values, including globbed paths e.g. `['/admin', 'api/**/update']`. Environment Variable: `DD_TRACE_CACHED_PAGES` , accepts an csv formatted string. *Experimental* | `[]` |
```

Comment:
I was thinking that `rum_injection_disabled_paths` would be a more intuitive name, give the main option is called `rum_injection_enabled`. Also, the reason the user wants to disable RUM on these paths could be unrelated to caching, even though caching is likely the main reason.

What do you think?

Reply:
After talking with @ericmustin, we decided to keep these names as is, as they match what other Datadog tracer languages are implementing it.